### ISSUE TITLE
John

### ISSUE THREAD

**Issue by chisel182** [PeterLinX/Introduction-to-Neo#11]
I fear I may have made a "rookie error." I fail to back up of course.
![neo](https://user-images.githubusercontent.com/30146557/29490552-fd2f0612-857e-11e7-8734-e3f4fe373b40.jpg)

Soon after I had installed the new NEO 2.0 wallet, then transfered 25 NEO's to it.

Then I had computer issues. NOE will not open. I their any specific file in the NEO directory/folder that I can extract my private keys. Or at best get NEO to execute properly.

**Comment by Soypedro** [comment]
Hi Chisel182, 

I have the similar problem right now. Is you'res solved? 

Regards, Peter

**Comment by chisel182** [comment]
No! It is difficult to find the support and information. Nobody replies to
emails, wallets don't download properly, don't finalize properly.

**Comment by Soypedro** [comment]
So you still cant access your'e Wallet?
And havent found anyone to get in contact with... ? That doesnt sound good.

Have you tried the official Neo helpdesk already? They might now where to find the right person to help..

Regards,
Peter Hagedoorn

Op 18 nov. 2017 om 00:41 heeft chisel182 <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:

No! It is difficult to find the support and information. Nobody replies to
emails, wallets don't download properly, don't finalize properly.
